### PR TITLE
Dockerize and update .gitignore

### PR DESCRIPTION
**Pull request by alexandrebis**
Still need to test a route

### CODE REVIEW

**Comment by alexandrebis** [comment]
Removed .idea/ in last commit

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM tomcat
+COPY /target/server-0.0.1-SNAPSHOT.war /usr/local/tomcat/webapps/jee-server.jar
```
Comment (paulinerouvel) [inline Dockerfile:2]: .war et  .jar ?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM tomcat
```
Comment (paulinerouvel) [inline Dockerfile:1]: Maybe add some documentation for dev who wants to launch the image ?

Comment (alexandrebis) [reply Dockerfile]: Done

File: Dockerfile (hunk 1 above)
Comment (alexandrebis) [reply Dockerfile]: Fixed in last commit #75a1f39

**Pull request merged**